explorer/types/explorertypes.go

```diff
@@ -390,15 +390,17 @@ func (mpi *MempoolInfo) Trim() *TrimmedMempoolInfo {
 	mpi.RUnlock()
 
 	// Calculate total fees for all mempool transactions.
-	getTotalFee := func(txs []*TrimmedTxInfo) (total float64) {
+	getTotalFee := func(txs []*TrimmedTxInfo) dcrutil.Amount {
+		var sum dcrutil.Amount
 		for _, tx := range txs {
-			total += tx.Fees
+			sum += tx.TxBasic.Fee
```

Comment:
NOTE: We can do this as a consequence of https://github.com/decred/dcrdata/pull/966